packages/wrapped-keys/src/lib/api/batch-generate-private-keys.ts

```diff
@@ -41,34 +41,40 @@ export async function batchGeneratePrivateKeys(
     pkpSessionSigs,
   });
 
-  const results = await Promise.all(
-    actionResults.map(
-      async (result): Promise<BatchGeneratePrivateKeysActionResult> => {
-        const { generateEncryptedPrivateKey, network } = result;
+  const keyParamsBatch = actionResults.map((keyData) => {
+    const { generateEncryptedPrivateKey, network } = keyData;
+    return {
+      ...generateEncryptedPrivateKey,
+      keyType: getKeyTypeFromNetwork(network),
```

Comment:
This is for Solana/EVM?

Reply:
Yep! it takes in the network string that is user-provided (evm or solana), and returns the type of key for that network (ed25519 or K256)